### DISCUSSION TITLE
Chiadog saying received 2XCH, when only receiving .25XCH pool payout.

### DISCUSSION THREAD

**Discussion by BeepBo0** [martomi/chiadog#335] [Q&A] [answered]
Anyone else getting this? 
I have a split farm. Whenever I get a pool plot win and payout, it always looks like I got a full OG win. It even says I received 0.25 and 1.75 XCH like an OG plot, even though the only value I receive is the standard 0.25 XCH pool payout.
Its not the end of the world, just wondered if this is normal and other people are getting it too, or is it a bug on my end?

**Comment by pieterhelsen** [comment, marked as answer]
Hi, 

Yes, this is normal. This is because the 1.75 is sent to the pooling wallet before it is claimed by the pool.
So you are briefly the owner of the 1.75XCH. 

I'd have to check the exact wording in the log files, but I think that Chiadog will be unable to _automagically_ discern whether you are pooling or self-pooling.

**Reply by BeepBo0** [reply]
> Hi,
> 
> Yes, this is normal. This is because the 1.75 is sent to the pooling wallet before it is claimed by the pool. So you are briefly the owner of the 1.75XCH.
> 
> I'd have to check the exact wording in the log files, but I think that Chiadog will be unable to _automagically_ discern whether you are pooling or self-pooling.

Awesome. Thanks for the fast answer. It would be great to at least have a -1.75xch when it goes out again!
